server.js

```diff
@@ -4,10 +4,12 @@ const next = require("next");
 const Cookies = require("universal-cookie");
 const helmet = require("helmet");
 const compression = require("compression");
+const fs = require("fs");
 
 const { parseUserAgent } = require("detect-browser");
 
 const dev = process.env.NODE_ENV !== "production";
+const staging = process.env.STAGING !== undefined;
```

Comment:
don't you also have to add this to .circleci/config.yml and Dockerfile?

Reply:
When we deploy the staging version to Azure we can just use the dockerfile we're already using, and inject the STAGING variable through Azure.

Wait, but why didn't I need to add STAGING to the app.json file that Heroku uses? 🤔

Reply:
well, the review app says
```
2019-02-06T18:44:41.333291+00:00 app[web.1]: Production environment, using static file
```
while the staging app says
```
2019-02-06T18:14:21.062285+00:00 app[web.1]: Staging environment, downloading from Airtable
```
🤷‍♂️